xclim/testing/tests/test_precip.py

```diff
@@ -400,21 +401,27 @@ def test_days_over_precip_doy_thresh():
     out1 = atmos.days_over_precip_doy_thresh(pr, per)
     np.testing.assert_array_equal(out1[1, :, 0], np.array([81, 61, 69, 78]))
 
-    out2 = atmos.days_over_precip_thresh(pr, per, thresh="2 mm/d")
+    out2 = atmos.days_over_precip_doy_thresh(pr, per, thresh="2 mm/d")
     np.testing.assert_array_equal(out2[1, :, 0], np.array([81, 61, 66, 78]))
 
     assert "only days with at least 2 mm/d are counted." in out2.description
+    assert "[80]th percentile" in out2.attrs["description"]
+    assert "['1990-01-01', '1993-12-31'] period" in out2.attrs["description"]
+    assert "5 day(s)" in out2.attrs["description"]
 
 
 def test_days_over_precip_thresh():
     pr = open_dataset("ERA5/daily_surface_cancities_1990-1993.nc").pr
     per = pr.quantile(0.8, "time", keep_attrs=True)
+    per = PercentileDataArray.from_da(per, build_climatology_bounds(pr))
```

Comment:
This case is what user must do when their percentiles/quantiles were not computed with `percentile_doy` in order to have the proper metadata.
Without this wrapping, they will get `{unknown}th percentile` and similar placeholders in the indicators metadata.

Reply:
What does @huard  think? Is this too laborious?

Reply:
No, I think that's reasonable.